io.github.anirbandey1.ChatGPTElectron.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Anirbnan Dey -->
+<component type="desktop-application">
+  <id>io.github.anirbandey1.ChatGPTElectron</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>ChatGPT Electron</name>
+  <developer_name>Anirban Dey</developer_name>
+  <url type="homepage">https://github.com/anirbandey1/chatgpt-electron/</url>
+  <url type="bugtracker">https://github.com/flathub/io.github.anirbandey1.ChatGPTElectron/issues</url>
+  <url type="help">https://github.com/anirbandey1/chatgpt-electron</url>
+  <summary>Electron wrapper for ChatGPT website</summary>
+  <description>
+    <p>A desktop application for using ChatGPT</p>
+    <p>This is a electron wrapper for ChatGPT website, packaged into a Flatpak. This repackaging is not supported by OpenAI.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/anirbandey1/chatgpt-electron-assets/blob/main/screenshots/login-page.png</image>
```

Comment:
this is not the image. it will not work.

Reply:
ok fixing it

Reply:
fixed in [commit](https://github.com/flathub/flathub/pull/4305/commits/4269698cc36e0f458fd6f5bf3f017a86352d3af3)